### DISCUSSION TITLE
Welcome to starknet-foundry Discussions!

### DISCUSSION THREAD

**Discussion by cptartur** [foundry-rs/starknet-foundry#195] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ukpatic** [comment]
Create Wallet
First create an Argent X or Bravoos wallet
the wallet is not available on mobile so use a PC
 Bridge Funds to Starknet

Interact with the native bridge if you haven’t 

✨Bridge from #ethereum to #starknet at starkgate.starknet.io

✨Connect MetaMask and Argent wallet

✨ Set Amount You Want To Transfer, & Click Transfer, Then Confirm Transaction In MetaMask

 To see other bridges you can interact with go to 

starknet.io/en/ecosystem/b…

✨Go to layerswap.io 
✨select chain you are bringing from and to
✨impute amount 
✨enter your Argent X wallet address and continue the transaction prompts

 Explore Dapps building on Starknet

Swap token and provide liquidity 

Go to app.jediswap.xyz 
✨connect your argent wallet
✨select any token pair of your choice and swap

 ✨To provide liquidity Go to app.jediswap.xyz/#/add/ETH/0x05…

✨select the token pair you wish to provide liquidity for 

✨Click on Supply, Confirm Supply and approve transaction in you wallet

You can use other Dapps like Myswap etc all is on Dappland and $STRK ecosystem 

Go to starknet.io/en/ecosystem/d… and interact with Dapps that are already live on the blockchain 

And ensure to create at least $1k volume above
 
Buy and Sell/Mint Nfts

How to Mint an Nft?

🔆First go to Mintsquare.io/mint
🔆Connect your Argent wallet
🔆 Upload a picture 
🔆Give it a name and a description 
🔆Click the 'Mint' button.
🔆Confirm the mint in your Argent X wallet (this transaction costs gas)

How to Buy an nft

🔆Browse mintsquare.io/starknet check for an NFT you like
🔆select the 'Add to cart' button.
🔆Confirm the purchase in Argent X.

How to SELL an NFT on Mintsquare
🔆Go to your profile on mint square 
🔆Select the nft you wish to sell
🔆 List the NFT for sale.
⏰ Wait for a buyer.

 Play a Game

Go on briq.construction

✨Connect wallet
✨Click “Start building”

 Now to start building you need some tools
At the top you will see 3 icons those are why you need 

💠 The first tool by left is the “Place tool” tap on then move to your dashboard and left click to pace your Briqs

🖌️ The second one is the “Paint tool” use to change the color of the briqs to your preferred color 
Just tap on it, by your left you will see different color display click on your preferred color then move your cursor to your briqs and left click to change your briqs color

📝 The one is the “Erase tool” you use it to remove any briqs you don’t want from your dashboard

 You need just 10 briqs minimum to build 
Because the more the briqs the higher the cost fee

After you have finished placing or building your briqs to your specifications Click on “Mint”

Click “mint” again

Agree to terms and conditions 
Click “Buy and mint” Approve it in your wallet
